package.json

```diff
@@ -107,7 +107,7 @@
     "ms": "0.7.1",
     "name-all-modules-plugin": "1.0.1",
     "node-sass": "3.7.0",
-    "notifications-panel": "1.2.7",
+    "notifications-panel": "github:Automattic/notifications-panel#react16",
```

Comment:
This can be set to `2.0.0` now – I published this version based on the `react16` branch.

Reply:
👯 👯 👯 ❤️  👯 👯 👯